lib/exception_handling.rb

```diff
@@ -595,6 +596,8 @@ def send_exception_summary( exception_data, first_seen, occurrences )
         deliver(ExceptionHandling::Mailer.exception_notification(exception_data, first_seen, occurrences))
       end
     rescue StandardError, MailerTimeout => ex
+      exception_context = exception_data[:environment]
+      exception_or_string = exception_data[:error_string]
```

Comment:
Is this really an exception_or_string still?  I was assuming (emphasis on assuming) that by this point exception_data is of the filtered variety, rather than the more raw version.  And with a name like "error_string.  I hope that it is an error string.